modules/release-notes/pages/relnotes.adoc

```diff
@@ -1,6 +1,100 @@
 = Release Notes for Couchbase Server 7.1
 :description: pass:q[Couchbase Server 7.1 introduces multiple new features.]
 
+[#release-711]
+== Release 7.1.1 (July 2022)
+
+Couchbase Server 7.1.1 was released in July 2022.
+This maintenance release contains fixes to issues.
+
+=== Fixed Issues
+
+This release contains the fixes listed below.
+
+=== Data Service
+
+[#table-known-issues-711-dataservice, cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-52248[MB-52248^]
+| Memcached hangs when no password is passed for encrypted private key
+
+|===
+
+=== XDCR
+
+[#table-known-issues-711-xdcr, cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-51939[MB-51939^]
+| XDCR does not update memcached flag/body after txn xattribute removal if user xattr DNE
+
+|===
+
+=== Query Service
+
+[#table-known-issues-711-queryservice, cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-52413[MB-52413^]
+| Negative integer in the 64bit range causes rounding
+
+| https://issues.couchbase.com/browse/MB-52255[MB-52255^]
+| Stop session hangs
+
+| https://issues.couchbase.com/browse/MB-52178[MB-52178^]
+| IN/NOT IN filters not using Hash for evaluation - continued
+
+| https://issues.couchbase.com/browse/MB-52179[MB-52179^]
+| LEFT JOIN breaks with between operator on non-existing attribute
+
+| https://issues.couchbase.com/browse/MB-52412[MB-52412^]
+| WITH clause distribution over union queries deviates from sql standard
+
+| https://issues.couchbase.com/browse/MB-52161[MB-52161^]
+| Adhoc query index selection issue with LIKE as index condition and query parameters
+
+| https://issues.couchbase.com/browse/MB-52511[MB-52511^]
+| Refresh_cluster_map fails with ERROR 199 :  N1QL: Invalid query service endpoint
+
+|===
+
+=== Eventing Service
+
+[#table-known-issues-711-eventingservice, cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-52492[MB-52492^]
+| Function causing recursion is missing from ERR_INTER_BUCKET_RECURSION error description
+
+|===
+
+=== Analytics Service
+
+[#table-known-issues-711-analyticsservice, cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-52008[MB-52008^]
+| Analytics average ingestion rate with magma is 90% lower compared to Couchstore
```

Comment:
Should remove this item.  Analytics wasn't certified to use magma in previous releases, so this isn't something that customers need to be aware of.

Reply:
I've removed.